NSS-Day-Cohort-49/giffygram-double-rainbows
# Favorite posts

#### Changes Made
1. Modified file `postlist.js` to include favorite button functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout favorite-posts
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > When verified user click "favorite" star button it populates the "likes" array within the json server and button allows for deselect to delete array info from json server 
    > Example: When user does BLANK, then BLANK should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
